cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -794,6 +795,162 @@ const char* gdv_fn_initcap_utf8(int64_t context, const char* data, int32_t data_
   *out_len = out_idx;
   return out;
 }
+
+GANDIVA_EXPORT
+const char* gdv_fn_elt_int32_utf8_utf8(int64_t context, int32_t pos, const char* word1,
+                                       int32_t word1_len, const char* word2,
+                                       int32_t word2_len, int32_t* out_len) {
+  const char* selected;
+  if (pos < 1 || pos > 2) {
+    *out_len = 0;
+    return "";
```

Comment:
This should return null. With this implementation, it will return a string of length 0. Please check NullInternal functions

Reply:
Sure, fixed.